packages/synapse-react-client/src/lib/containers/auth/TwoFactorAuthSettingsPanel.tsx

```diff
@@ -0,0 +1,100 @@
+import { Box, Button, Typography, Stack, Skeleton } from '@mui/material'
+import React from 'react'
+import IconSvg from '../IconSvg'
+import { TWO_FACTOR_DOCS_LINK } from './TwoFactorEnrollmentForm'
+import {
+  useDisableTwoFactorAuth,
+  useGetTwoFactorEnrollmentStatus,
+} from '../../utils/hooks/SynapseAPI/auth/useTwoFactorEnrollment'
+import ConditionalWrapper from '../utils/ConditionalWrapper'
+import { displayToast } from '../ToastMessage'
+
+export type TwoFactorAuthSettingsPanelProps = {
+  onRegenerateBackupCodes: () => void
+  onBeginTwoFactorEnrollment: () => void
+}
+
+export default function TwoFactorAuthSettingsPanel(
+  props: TwoFactorAuthSettingsPanelProps,
+) {
+  const { onRegenerateBackupCodes, onBeginTwoFactorEnrollment } = props
+  const { data: status, isLoading: isLoadingStatus } =
+    useGetTwoFactorEnrollmentStatus()
+  const isActivated = status?.status === 'ENABLED'
+
+  const { mutate: disable2FA, isLoading: isLoadingMutation } =
+    useDisableTwoFactorAuth({
+      onSuccess: () => {
+        displayToast('2FA removed from this account', 'info')
+      },
+    })
+
+  const isLoading = isLoadingStatus || isLoadingMutation
+
+  return (
+    <Box>
+      <Typography variant={'headline2'}>
+        Two-factor Authentication (2FA)
+      </Typography>
+      <ConditionalWrapper condition={isLoading} wrapper={Skeleton}>
+        <Typography
+          variant={'body1'}
+          color={isActivated ? 'success.main' : 'error.main'}
+          sx={{ my: 2, display: 'flex', alignItems: 'center' }}
+        >
+          <IconSvg
+            icon={isActivated ? 'check' : 'cross'}
+            sx={{ mr: 1, height: '24px' }}
+          />
+          {isLoading ? 'Loading...' : isActivated ? 'Active' : 'Inactive'}
```

Comment:
`'Loading...'` is never seen but it makes it easier to right warning-free tests, since the 'Active'/'Inactive' text won't be queryable until the API call completes.